onnx/defs/schema.h

```diff
@@ -558,6 +560,14 @@ class OpSchema final {
     return tensor_inference_function_ ? true : false;
   }
 
+  bool has_function_body() const {
+    return function_body_.node_size() > 0;
+  }
+
+  OpSchema& FunctionBody(std::vector<NodeProto>& func_nodes);
+
+  const FunctionProto* GetFunctionBody() const;
```

Comment:
a better name  is GetFunciton, as here's not only getting its body (bunch of nodes), but all function related stuff.